src/views/AddItem.jsx

```diff
@@ -1,3 +1,90 @@
-export function AddItem() {
-	return <p>Hello from the <code>/add-item</code> page!</p>
+import { addItem } from '../api/firebase';
+import { useState } from 'react';
+
+export function AddItem({ listId }) {
+	const [itemData, setItemData] = useState({
+		itemName: '',
+		daysUntilNextPurchase: null,
+	});
+	/*
+	itemData = {
+		itemName:
+		daysUntilNextPurchase:
+	}
+	*/
+
+	const onFormSubmit = (event) => {
+		event.preventDefault();
+		addItem(listId, itemData);
+		console.log(itemData);
```

Comment:
when this logs, it seems like the data payload is incomplete. when I tested, it only had the daysUntilNextPurchased, and I think that's the last form field i had filled out.